packages/components/tests/dummy/app/templates/components/pagination-bar.hbs

```diff
@@ -64,9 +72,9 @@
 <section data-test-percy>
   <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
 
-  {{! ADD YOUR CONTENT HERE }}
-
-  {{! This below is just an example of invocation, to get started }}
+  <h4 class="dummy-h4">Default (numbered)</h4>
   <Hds::PaginationBar />
 
+  <h4 class="dummy-h4">Compact</h4>
+  <Hds::PaginationBar class="hds-pagination-bar--is-compact" />
 </section>
```

Comment:
The docs are all just preliminary. Will clean up, modify, and expand upon them as I continue...